app/controllers/physical_server_controller.rb

```diff
@@ -0,0 +1,117 @@
+class PhysicalServerController < ApplicationController
+  include Mixins::GenericListMixin
+  include Mixins::GenericShowMixin
+
+  before_action :check_privileges
+  before_action :session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  def self.table_name
+    @table_name ||= "physical_servers"
+  end
+
+  def session_data
+    @title  = _("Physical Servers")
+    @layout = "physical_server"
+    @lastaction = session[:physical_server_lastaction]
+  end
+
+  def collect_data(server_id)
+    PhysicalServerService.new(server_id, self).all_data
+  end
+
+  def set_session_data
+    session[:layout] = @layout
+    session[:physical_server_lastaction] = @lastaction
+  end
+
+  def show_list
+    # Disable the cache to prevent a caching problem that occurs when
+    # pressing the browser's back arrow button to return to the show_list
+    # page while on the Physical Server's show page. Disabling the cache
+    # causes the page and its session variables to actually be reloaded.
+    disable_client_cache
```

Comment:
@martinpovolny We ran into the following issue:

1. On the page that displays multiple physical servers (show_list), click on the quadicon of a server. It should now take you to the summary page for that physical server (show).
2. Click on the browser's back button to return to the previous page (show_list).
3. Now select the checkbox for one of the physical servers. Notice that even though a server is selected, the "Identify" and "Power" drop downs and associated buttons are not selectable (they are grayed-out).

This behavior is due to session variables not being updated when clicking on the browser's back button. When the previous page is reloaded, only the cached values for the variables are used.